op-challenger/game/fault/trace/vm/kona_interop_server_executor.go

```diff
@@ -0,0 +1,48 @@
+package vm
+
+import (
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/trace/utils"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type KonaInteropExecutor struct {
+	nativeMode     bool
+	agreedPreState []byte
+}
+
+var _ OracleServerExecutor = (*KonaInteropExecutor)(nil)
+
+func NewKonaInteropExecutor(agreedPreState []byte) *KonaInteropExecutor {
```

Comment:
nit: SuperExecutor?

I don't really mind either way.

Reply:
yeah not opinionated either, super executor it is